c7n/reports/csvout.py

```diff
@@ -155,6 +167,15 @@ def __init__(
             # TODO this type coercion should be done at cli input, not here
```

Comment:
@kapilt - unrelated to this commit, but I just noticed this comment.  It is doing type-checking in `cli.py` to ensure that this is key=val pairs.  (see `_key_val_pair` function in `cli.py`)